src/Moq/IInvocation.cs

```diff
@@ -35,5 +36,13 @@ public interface IInvocation
 		/// Gets the return value of the invocation.
 		/// </summary>
 		object ReturnValue { get; }
```

Comment:
We may add a `remark` her too:
```suggestion
		/// <remarks>
		/// Returns <see langword="null" /> if the invocation did throw an <see cref="Exception" /> or was 
		/// a <see langword="void" /> method.
		/// </remarks>
		object ReturnValue { get; }
```

Reply:
Done 988feb05 (I like the Moq 5 comments!)